### PR TITLE
Make the annotations (attributes) read by DualReader unique

### LINKED ISSUE #13: Make the annotations (attributes) read by DualReader unique
With Dual Reader, two "annotations" will be read in the following cases.

```php
#[FakeCacheable]
/**
 * @FakeCacheable
 */
class FakeDual
```

Shouldn't only one annotation be read?
(This can be done using `array_unique`)

It is mainly intended for library code, so all users of the doctrine/annotation reader, attribute reader, and the dual reader will get the same results.

Upgrading to 1.0.1 has caused [some tests to fail](https://github.com/bearsunday/BEAR.Package/runs/3053439593?check_suite_focus=true) .

@TomasVotruba Thoughts?

### PR DESCRIPTION
**Pull request by koriym**
Closes #13

### CODE REVIEW

File: src/DualReader.php [hunk 1]
```diff
@@ -78,7 +80,7 @@ public function getPropertyAnnotations(ReflectionProperty $property): array
 
         $attributes = $this->attributeReader->getPropertyAnnotations($property);
 
-        return array_merge($annotations, $attributes);
+        return array_unique(array_merge($annotations, $attributes), SORT_REGULAR);
```
Comment (TomasVotruba) [inline src/DualReader.php:83]: Would this keep same annotation that only differ in one parameter?

Comment (koriym) [reply src/DualReader.php:83]: Yes, test for it added.
@see https://github.com/koriym/Koriym.Attributes/pull/14/commits/815149331c9a545366eaf83508b1de1cb67ee80a

Comment (TomasVotruba) [reply src/DualReader.php:83]: Awesome 😎👍

Comment (koriym) [reply src/DualReader.php:83]: Will be released tomorrow.😎

Comment (TomasVotruba) [reply src/DualReader.php:83]: Thank you! Have a good night sleep 😴

**Pull request merged**